cpp/example_code/s3/create_bucket.cpp

```diff
@@ -1,96 +1,75 @@
- 
-//snippet-sourcedescription:[create_bucket.cpp demonstrates how to create an Amazon S3 bucket.]
-//snippet-service:[s3]
-//snippet-keyword:[Amazon S3]
-//snippet-keyword:[C++]
-//snippet-sourcesyntax:[cpp]
-//snippet-keyword:[Code Sample]
-//snippet-sourcetype:[snippet]
-//snippet-sourcedate:[2019-06-20]
-//snippet-sourceauthor:[AWS]
+﻿// Copyright Amazon.com, Inc. or its affiliates.All Rights Reserved.
+// SPDX - License - Identifier: Apache - 2.0
 
-/*
-   Copyright 2010-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
-
-   This file is licensed under the Apache License, Version 2.0 (the "License").
-   You may not use this file except in compliance with the License. A copy of
-   the License is located at
-
-    http://aws.amazon.com/apache2.0/
-
-   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
-   CONDITIONS OF ANY KIND, either express or implied. See the License for the
-   specific language governing permissions and limitations under the License.
-*/
-
-//snippet-start:[s3.cpp.create_bucket.inc]
+// snippet-start:[s3.cpp.create_bucket.inc]
+#include <iostream>
 #include <aws/core/Aws.h>
 #include <aws/s3/S3Client.h>
 #include <aws/s3/model/CreateBucketRequest.h>
-//snippet-end:[s3.cpp.create_bucket.inc]
+#include <aws/s3/model/BucketLocationConstraint.h>
+// snippet-end:[s3.cpp.create_bucket.inc]
+#include <awsdoc/s3/s3_examples.h>
 
-/**
- * Create an Amazon S3 bucket in a specified region
- */
-// snippet-start:[s3.cpp.create_bucket.code]
-bool create_bucket(const Aws::String &bucket_name,
-    const Aws::S3::Model::BucketLocationConstraint &region = Aws::S3::Model::BucketLocationConstraint::us_east_1)
+/* ////////////////////////////////////////////////////////////////////////////
+ * Purpose: Creates a bucket in Amazon S3.
+ *
+ * Inputs:
+ * - bucketName: The name of the bucket to create. 
+ * - region: The AWS Region to create the bucket in.
+ * 
+ * Outputs: true if the bucket was created; otherwise, false.
+ * ///////////////////////////////////////////////////////////////////////// */
+
+ // snippet-start:[s3.cpp.create_bucket.code]
+bool AwsDoc::S3::CreateBucket(Aws::String bucketName, 
+        Aws::S3::Model::BucketLocationConstraint region)
 {
-    // Set up the request
+	Aws::S3::S3Client s3_client;

```

Comment:
https://github.com/aws/aws-sdk-cpp#contributing-back
Most of naming in AWS SDK for C++ are using cammelCase. But I think that's fine to have a different formatting in examples.